lib/components/SModal.vue

```diff
@@ -16,12 +16,12 @@ const emit = defineEmits<{
 
 const el = ref<any>(null)
 
-function onClick(e: any) {
+function onClick(e: MouseEvent) {
   if (!props.closable) {
     return
   }
 
-  if (e.target === el.value) {
+  if (e.button === 0 && e.target === el.value) {
```

Comment:
Just curious, what is `e.button`? 👀  Never used it before.

Reply:
primary click (generally left button)

Reply:
Ah, gotcha! Nice to know 👍